SECURITY.md

```diff
@@ -0,0 +1,194 @@
+# Security Policy
+
+## Supported Versions
+
+| Version             | Supported          |
+| -------             | ------------------ |
+| 0.28                | :white_check_mark: |
+| 0.27.4              | :white_check_mark: |
+| 0.27.3 and earlier  | :x:                |
+
+## The security release process
+
+Security issues are reported as issues on https://github.com/exiv2/exiv2 with the label Security.
+
+1.) Someone should feel responsible and make sure that it's really a security
+    defect. If yes, we need one person who is responsible for this specific
+    issue beyond this point. This person coordinates the further actions and
+    might delegate tasks, of course.
+
+2.) Create a bug report with limited access to "libssh core developers" only!
```

Comment:
There is a mention here to **libssh**. I guess we need to adapt the paragraph to Exiv2